src/main/kotlin/org/rust/lang/core/psi/impl/mixin/RustImplMethodMemberImplMixin.kt

```diff
@@ -25,10 +25,7 @@ abstract class RustImplMethodMemberImplMixin : RustFnImplMixin<RustImplMethodMem
     override fun getParent(): PsiElement? = parentByStub
 
     override fun getIcon(flags: Int): Icon? {
-        var icon = RustIcons.METHOD
-        if (isStatic) {
-            icon = icon.addStaticMark()
-        }
+        val icon = if (isStatic) RustIcons.FUNCTION.addStaticMark() else RustIcons.METHOD
```

Comment:
It totally makes sense. I didn't find any predefined icons like that in the existing code, so decided to implement it the same way as other similar classes do. I'll extract the icon.